tools/dependencydetect.js

```diff
@@ -0,0 +1,23 @@
+const glob = require('glob');
+const fs = require('fs');
+const folder = '../servers/**/';
+
+glob(`${folder}package.json`, null, (err, files) => {
+    if (err) return console.log('Unable to scan directory: ' + err);
+
+    files.forEach(file => {
+        fs.readFile(file, 'utf-8', (err, data) => {
+            if (err) return console.log('Unable to scan directory: ' + err);
+            const obj = JSON.parse(data);
+            const { dependencies } = obj;
+            for (let key in dependencies) {
+                if (dependencies[key].includes('file')){
+                    dependencies[key] = '^0.0.1';
```

Comment:
Should not hardcode the version, instead it should it need pull it from the resolved file path of the package.jason. 

For example 

```
"@sample-stack/core": ""file:../../packages/sample-core"",
```

to 

```
"@sample-stack/core": "^0.0.39-alpha.9",
```
Where `0.0.39-alpha.9` is from [here](https://github.com/cdmbase/fullstack-pro/blob/testpublish/packages/sample-core/package.json#L3)

So `testpublish` branch will have versions of the packages that server's package.json file need to be updated.

Reply:
The goal is, when we are in `testdevelop` we don't have version of those packages, but as we switch to `testpublish` branch we will use this script to update the server's package.json to resolved versions of each dependency.